test/runtests.jl

```diff
@@ -27,6 +27,12 @@ end
 Polynomial(args::T...) where {T<:Real} = Polynomial(collect(args))
 (p::Polynomial)(x) = sum(a * x^(i-1) for (i, a) in enumerate(p.coeffs))
 
+function testmissing(vec1, vec2)
+  r1 = map(!ismissing, vec1) == map(!ismissing, vec2)
+  r2 = findall(ismissing, vec1) == findall(ismissing, vec2)
+  r1 && r2
```

Comment:
Can you elaborate on why we need these two checks instead of just the last with findall?

Reply:
This test pass without first check:
```julia
[missing, 1, 2] == [missing, 2, 1]
```
And this test pass without second check:
```julia
[missing, 1, 2] == [1, 2, missing]
```

Reply:
Thanks, I guess we could rewrite this helper function with a for loop like in the previous PR where we first check if the two entries are not missing and then compare the values?

Reply:
Sorry, I messed up. Is no `map` is `filter`.

Reply:
Resolved with [0a3319d](https://github.com/JuliaML/TableTransforms.jl/pull/28/commits/0a3319d91a86cd4310f326976725eb82f7fadba2)